src/index.ts

```diff
@@ -408,7 +408,7 @@ async function launchMetals(
           makeVimDoctor(JSON.parse(doctorJson));
           break;
         case ClientCommands.ReloadDoctor:
-          workspace.nvim.call("coc#util#has_preview").then((preview) => {
+          workspace.nvim.call("coc#list#has_preview").then((preview) => {
```

Comment:
Since we have this actually being exported from commands, can we just use `Commands.HAS_PREVIEW` here?
```suggestion
          workspace.nvim.call(Commands.HAS_PREVIEW).then((preview) => {
```

Reply:
Done.